tests/csapi/rooms_members_local_test.go

```diff
@@ -4,20 +4,55 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/tidwall/gjson"
+
 	"github.com/matrix-org/complement/internal/b"
 	"github.com/matrix-org/complement/internal/client"
-	"github.com/matrix-org/complement/runtime"
-	"github.com/tidwall/gjson"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
 )
 
 func TestMembersLocal(t *testing.T) {
-	deployment := Deploy(t, b.BlueprintOneToOneRoom)
+	deployment := Deploy(t, b.MustValidate(b.Blueprint{
```

Comment:
Can we please use `BlueprintOneToOneRoom`?

Reply:
No, because that messes with the presence test, as then Alice already shares a room with Bob, and it's not conclusively testing if a new sync will include *newly seen* members' presence.

If we use one to one, the latter condition isn't true anymore, as Alice has already "seen" Bob before, in another room.

Reply:
I'll change it to BlueprintAlice and register bob afterwards